packages/checkout-storefront/rollup.config.js

```diff
@@ -4,48 +4,47 @@ import typescript from "@rollup/plugin-typescript";
 import { terser } from "rollup-plugin-terser";
 import external from "rollup-plugin-peer-deps-external";
 import postcss from "rollup-plugin-postcss";
-import dts from "rollup-plugin-dts";
 import json from "@rollup/plugin-json";
 import image from "@rollup/plugin-image";
 
 const packageJson = require("./package.json");
 
+const isDev = process.env.NODE_ENV !== "production";
```

Comment:
Since everywhere the output of `isDev` is negated maybe we should do `isProd`? 

```suggestion
const isProd = process.env.NODE_ENV === "production";
```